<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" rowspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; background-color:transparent;&quot;&gt;The current optimization screen will be displayed here. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; background-color:transparent;&quot;&gt;Normally, the content of each well is displayed as the concentration of each in the well plus the volume required to produce that concentration given its stock concentration and the well volume. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; background-color:transparent;&quot;&gt;However, HWI cocktail data sheets use a variety of units to describe reagent concentrations within a cocktail. Namely, % w/v and % v /v. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; background-color:transparent;&quot;&gt;In order to convert to morality for these units, one must know the molar mass or molar mass and density respectively. These values are unknown for a subset of reagents and therefore calculating a volume of reagent may not always be possible. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; background-color:transparent;&quot;&gt;Please note all units when reviewing a screen.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_17">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Plate Setup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Plate Dimensions</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_2">
        <property name="toolTip">
         <string>Change the x (number of rows) dimension of your screening plate.</string>
        </property>
        <property name="maximum">
         <number>1536</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_34">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the volume of an individual well in your optimization plate.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Well Volume</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="UnitComboBox" name="unitComboBox_2" native="true">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the units to display reagent volumes in.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Output Units</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QSpinBox" name="spinBox_3">
        <property name="toolTip">
         <string>Change the y (number of columns) dimension of your screening plate.</string>
        </property>
        <property name="maximum">
         <number>1536</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QComboBox" name="comboBox_16">
        <property name="toolTip">
         <string>Set the stock solution unit in the final output.</string>
        </property>
        <item>
         <property name="text">
          <string>ul</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ml</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>l</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_27">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>350</height>
      </size>
     </property>
     <property name="title">
      <string>Reagents Controls</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>Hit Well</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_12">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Well numbers of images you have classified as crystal containing. MARCO classifications are not included. Pick from these wells to optimize.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_11">
         <property name="toolTip">
          <string>Set the reagent to be varried on the x-axis of the plate</string>
         </property>
         <attribute name="title">
          <string>X Reagent</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Assign Reagent</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_38">
            <property name="text">
             <string>Varry each well by</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
            <property name="toolTip">
             <string>Percent difference of each well along the x axis in reference to the x reagent hit concentration.</string>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_39">
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>Stock Concentration</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="UnitComboBox" name="unitComboBox" native="true">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the stock concentration of your selected x reagent&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QComboBox" name="comboBox_6">
            <property name="toolTip">
             <string>Select the reagent to vary across the x axis of the plate.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_12">
         <property name="toolTip">
          <string>Set the reagent to be varried in the y-axis of the plate</string>
         </property>
         <attribute name="title">
          <string>Y Reagent</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>Assign Reagent</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>Vary each well by</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
            <property name="toolTip">
             <string>Percent difference of each well along the y axis in reference to the y reagent hit concentration.</string>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_42">
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_43">
            <property name="text">
             <string>Stock Concentration</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="UnitComboBox" name="unitComboBox_3" native="true">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the stock concentration of your selected y reagent.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QComboBox" name="comboBox_13">
            <property name="toolTip">
             <string>Select the reagent to vary on the y axis of the plate.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_13">
         <attribute name="title">
          <string>Constants</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_36">
            <property name="text">
             <string>Constant Reagents</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QListWidget" name="listWidget_4">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reagents that have not been selected as the x nor y reagent appear hear.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_37">
            <property name="text">
             <string>Stock Concentration</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="UnitComboBox" name="unitComboBox_4" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_44">
            <property name="text">
             <string>pH</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
            <property name="toolTip">
             <string>Set the pH of your screen across the optimization plate.</string>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_26">
        <property name="toolTip">
         <string>Export the current optimization screen to an html file. Will open a file browser and ask you to specify a location and filename for your export.</string>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_27">
        <property name="toolTip">
         <string>Render the optimization plate to the screen.</string>
        </property>
        <property name="text">
         <string>Show Screen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UnitComboBox</class>
   <extends>QWidget</extends>
   <header>polo/widgets/unit_combo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
